README.md

```diff
@@ -1,15 +1,15 @@
-# Simulator project for LVGL embedded GUI Library
+# Simulator project for LVGL embedded GUI Library running on FreeRTOS
 
 The [LVGL](https://github.com/lvgl/lvgl) is written mainly for microcontrollers and embedded systems, however you can run the library **on your PC** as well without any embedded hardware. The code written on PC can be simply copied when your are using an embedded system.
 
-This project is configured for [VSCode](https://code.visualstudio.com) and only tested on Linux, although this may work on OSx or WSL. It requires a working version of GCC, GDB and make in your path.
+## (RT)OS support
+Works with any OS like pthred, Windows, FreeRTOS, etc. It has build in support for FreeRTOS. 
```

Comment:
This section shouldn't be required here.

Reply:
I removed this section and placed it at the end with a small hint.